domains/certificates/Query.API/API.IntegrationTests/Repositories/SlidingWindowStateTests.cs

```diff
@@ -41,7 +41,7 @@ public SlidingWindowStateTests(IntegrationTestFixture integrationTestFixture)
     [Fact]
     public async Task GetSlidingWindowStartTime_NoDataInStore_ReturnsContractStartDate()
     {
-        var info = CreateSyncInfo();
+        var info = CreateSyncInfo(GsrnHelper.GenerateRandom());
```

Comment:
I would delete GSRN Helper and move logic into GSRN. Like guids. Guid.NewGuid() GSRN.NewGsrn()

Reply:
I don't agree since we are not supposed to create new random GSRNs in production code

Reply:
Ahhh :D Didn't see this was for test only.